### ISSUE TITLE
Build error in Example

### ISSUE THREAD

**Issue by ffittschen** [Skyscanner/backpack-ios#1331]
Hi there, I'd like to run your Example app, but unfortunately there is a build error (see attached screenshot).

> /Users/user/Code/backpack-ios/Example/Backpack/Backpack Native-Bridging-Header.h:23:9: error: 'BPKThemeContainerController.h' file not found
#import "BPKThemeContainerController.h"
        ^
1 error generated.
<unknown>:0: error: failed to emit precompiled header '/Users/user/Library/Developer/Xcode/DerivedData/Backpack-bhocfupkndphmcamwrekbprejfnl/Build/Intermediates.noindex/PrecompiledHeaders/Backpack Native-Bridging-Header-swift_2S3Q792V1CMPG-clang_2XBZU5UVWZVSQ.pch' for bridging header '/Users/user/Code/backpack-ios/Example/Backpack/Backpack Native-Bridging-Header.h'
2 errors generated.

<img width="450" alt="Screen Shot 2022-06-15 at 12 05 34@2x" src="https://user-images.githubusercontent.com/7734806/173802460-1ea2b331-073d-4da5-9eb3-97399e2a18f7.png">

I ran all steps outlined in the [Setup section](https://github.com/Skyscanner/backpack-ios/blob/main/CONTRIBUTING.md#setup) of the contributing guidelines and I'm using Xcode 13.4 on macOS 12.4 with the latest `main` of this repo.

**Comment by gert-janvercauteren** [comment]
Good morning @ffittschen 
We have prepared a pull request that should unblock building the example app on M1

https://github.com/Skyscanner/backpack-ios/pull/1341
In the mean-time, you can remove 'excludedArchs' from the build settings.

We expect the PR to be merged later today though :D

**Comment by ffittschen** [comment]
I would have never looked at that build setting based on this build error... Thanks for the PR @gert-janvercauteren. There was still another `EXCLUDED_ARCHS` build setting on the target level, so I created a follow-up PR: https://github.com/Skyscanner/backpack-ios/pull/1366
